ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -657,7 +663,7 @@
     </layout>
   </citation>
   <!--_-_-_-_-_-SECTION DEFINITION DE LA BIBLIO (bibliography)-_-_-_-_-_-_-->
-  <bibliography>
+  <bibliography  hanging-indent="true" entry-spacing="2">
```

Comment:
are you sure about entry-spacing="2"? That's two (!) line breaks between entries. 

Entry 1
\
\
Entry 2